tests/addons/test_config.py

```diff
@@ -241,3 +248,7 @@ def test_watchdog_url():
     ):
         config["watchdog"] = test_options
         assert vd.SCHEMA_ADDON_CONFIG(config)
+
+
+async def test_addon_build_from():
+    """Test build per arch or manifest approach."""
```

Comment:
empty test?

Reply:
oh yea whoops, I started that then found the already existing one to test the normal `build_from`. Thanks